core/src/test/scala/integration/kafka/api/TransactionsTest.scala

```diff
@@ -301,12 +300,11 @@ class TransactionsTest extends IntegrationTestHarness {
     assertEquals(3L, second.offset)
   }
 
-  @nowarn("cat=deprecation")
   @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
   @MethodSource(Array("getTestQuorumAndGroupProtocolParametersClassicGroupProtocolOnly_KAFKA_17961"))
   def testSendOffsetsWithGroupId(quorum: String, groupProtocol: String): Unit = {
     sendOffset((producer, groupId, consumer) =>
-      producer.sendOffsetsToTransaction(TestUtils.consumerPositions(consumer).asJava, groupId))
+      producer.sendOffsetsToTransaction(TestUtils.consumerPositions(consumer).asJava, consumer.groupMetadata()))
```

Comment:
Please remove this duplicate test case :)

Reply:
> Please remove this duplicate test case :)

Okay.